_posts/2021-08-26-geometric-deep-learning/geometric_deep_learning.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: "Beyond alchemy: A first look at geometric deep learning"
+description: > 
+ Geometric deep learning is a "program" that aspires to situate deep learning architectures and techniques in a framework of mathematical priors. The priors, such as various types of invariance, first arise on some physical domain. A neural network that well matches the domain will preserve as many invariances as possible. In this post, we present a very conceptual, high-level overview, and highlight a few applications.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanageometricdl
+date: 08-26-2021
+bibliography: bibliography.bib
+categories:
+  - Concepts
+  - Meta
+  - R
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/preview.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+To the practitioner, it may often seem that with deep learning, there is a lot of magic involved. Magic in how hyper-parameter choices affect performance, for example. More fundamentally yet, magic in the impacts of architectural decisions. Magic, sometimes, in that it even works. (Or not.) Sure, papers abound that strive to mathematically prove why, for specific solutions, in specific contexts, this or that technique will yield better results. But theory and practice are strangely dissociated: If a technique *does* turn out to be helpful in practice, doubts may still arise to whether that is, in fact, due to the purported mechanism. Moreover, level of generality often is low.
+
+In this situation, one may feel grateful for approaches that aim to elucidate, complement, or replace some of the magic. By "complement or replace", I'm alluding to attempts to incorporate domain-specific knowledge into the training process. Interesting examples exist in several sciences, and I certainly hope to be able to showcase a few of these, on this blog, at a later time. As to the "elucidate", this characterization is meant to lead on to the topic of this post: the program of *geometric deep learning*.
+
+## Geometric deep learning: An attempt at unification
+
+Geometric deep learning (henceforth: GDL) is what a group of researchers around Michael Bronstein, Joan Bruna, Taco Cohen, and Petar Velicković call their attempt to build a framework that places deep learning (DL) on a solid mathematical basis.
```

Comment:
What do you mean by "group of researchers around…"? Are the ones you listed not among those researchers? If they are, you might replace "around" with "including", or something to that effect.